exercises/bob/canonical-data.json

```
@@ -62,7 +62,7 @@
       "description": "using acronyms in regular speech",
       "property": "response",
       "input": {
-        "heyBob": "It's OK if you don't want to go to the DMV."
+        "heyBob": "It's OK if you don't want to go to the dentist."
```

This case no longer tests using an acronym in regular speech. 
```suggestion
        "heyBob": "It's OK if you don't want to go work for NASA."
```